src/birthder.js

```diff
@@ -236,8 +239,8 @@
    * Form a reminder message
    *
    * @param {Object} users - User object where each key is the user instance.
-   * @param {moment} targetDay - Date today
-   * @param {number} amountOfTime - Amount of time before birthday
+   * @param {moment} targetDay - Date today.
```

Comment:
I guess `Date today` is not the best description for the param. We need something different.